test/pylib/db/model.py

```diff
@@ -0,0 +1,26 @@
+from attrs import define, AttrsInstance
+
+
+@define
+class TestMetricRecord(AttrsInstance):
+
+    architecture: str = None
+    mode: str = None
+    test_name: str = None
+    run_id: int = None
+    directory: str = None
+    user_usec: int = None
+    system_usec: int = None
```

Comment:
Suggest to use floats with units of seconds, it's more natural.

Reply:
done